UPDATING.md

```diff
@@ -180,6 +180,11 @@ elasticsearch_end_of_log_mark = end_of_log
 ```
 
 The previous setting of `log_task_reader` is not needed in many cases now when using the default logging config with remote storages. (Previously it needed to be set to `s3.task` or similar. This is not needed with the default config anymore)
+basically 
+```
+[core]
+task_log_reader = task
```

Comment:
This section is not as clear as it can be. The word `basically` does not add any discernibility. Are you saying that the section you added is the new state or the old state? Please add more text to convey what the old and new states are with illustrative config section examples. Like 
```
before upgrade
   config section
after upgrade
   config section
```